ReadMe.md

```diff
@@ -1,4 +1,24 @@
-# Law Pokedex
+# Pokédex
+
+A live Pokédex web application in JavaScript, HTML and CSS and deployed using [Heroku app](https://www.heroku.com/).
+
+![image](https://user-images.githubusercontent.com/58405138/73666687-15edc280-469b-11ea-9ee4-8465515b3233.png)
+
+![image](https://user-images.githubusercontent.com/58405138/73878168-fe0c6f00-4851-11ea-8b05-679bb26f1185.png)
```

Comment:
You can make these images relative to the repo. I would recommend putting these in a folder called `.github` that way you can refer to these as ![image](.github/image1)

Also, I would like to see proper captions instead of just `image`